messages/en.json

```diff
@@ -52,6 +52,9 @@
 			"hour": "{count, plural, =1 {# hour} other {# hours}}",
 			"month": "{count, plural, =1 {# month} other {# months}}",
 			"week": "{count, plural, =1 {# week} other {# weeks}}"
+		},
+		"actions": {
+			"shopNow": "Magasinez maintenant"
```

Comment:
@service-paradis could you revert that to `Shop now` as this is the `en.json` file and move that translation to the French file?

Reply:
🤦‍♂️ Sure, sorry for this!